package.json

```diff
@@ -42,6 +44,7 @@
     "CHANGELOG.md"
   ],
   "scripts": {
+    "debug": "serve -n",
```

Comment:
Let's leave this out for now, should be fine just to open files on the local filesystem or use something like `npx http-server`.

Reply:
Ya, this was just meant as an example since ES modules cannot be loaded from the filesystem by web browsers.

Reply:
For sure, this needs to be resolved. The development flow in this regard is less than ideal at the moment. But let's tackle that another time.